src/extensions/actions/ActionsRuleCreationModal.tsx

```diff
@@ -191,11 +208,28 @@ export const NeoCustomReportActionsModal = ({
     return suggestions;
   };
 
+  const handleOnInputchange = (customization, index, value) => {
+    console.log(customization, index, value);
+    updateRuleField(index, 'value', value);
```

Comment:
Existing issue:
This does not work for 'go to page' actions on the bar chart.
Need to figure out how to fix.